docs/source/technical_tutorials/access/copc_access.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>This output doesn't make sense. Where did this come from? I'm pretty sure that <code>maap.searchGranule</code> doesn't generate this output, but perhaps I'm mistaken and it has recently changed to produce it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>